js/customizer.js

```diff
@@ -6,6 +6,29 @@
  * Contains handlers to make Theme Customizer preview reload changes asynchronously.
  */
 
-( function( $ ) {
+( function( value ) {
+	//Update site header overlay color...
+	wp.customize( 'site_header_overlay_color', function( value ) {
+		value.bind( function( newval ) {
 
-} )( jQuery );
+			var css = `
+				.site-header.featured-image .hentry:before,
+				.site-header.featured-image .hentry:after {
+					background: ${ newval };
+  				}
+  			`,
```

Comment:
We should avoid duplicating the CSS in the JavaScript file. My recommended approach is to do something like this:

`value.bind( function( to ) {

	// Update custom color CSS
	var style = $( '#twentyseventeen--color' ), // Selector of output style element.
	    color = style.data( 'color' ),
	    css = style.html();
	css = css.split( color ).join( to ); // equivalent to css.replaceAll.
	style.html( css )
	       .data( 'color', to );
} );

See https://celloexpressions.com/blog/a-strategy-for-custom-colors-in-the-customizer/ for the full explanation of this approach.